pkg/daemon/update.go

```diff
@@ -556,6 +587,122 @@ func (dn *Daemon) updateKernelArguments(oldConfig, newConfig *mcfgv1.MachineConf
 	return exec.Command("rpm-ostree", args...).Run()
 }
 
+// MountOSContainer mounts the container and returns the mountpoint
+func (dn *Daemon) MountOSContainer(container string) (mnt, containerName string, err error) {
+
+	var authArgs []string
+	if _, err = os.Stat(kubeletAuthFile); err == nil {
+		authArgs = append(authArgs, "--authfile", kubeletAuthFile)
+	}
+	// Pull the image
+	args := []string{"pull", "-q"}
+	args = append(args, authArgs...)
+	args = append(args, container)
+	pivotutils.RunExt(false, numRetriesNetCommands, "podman", args...)
```

Comment:
can we check this is run successfully ?

Reply:
according to: https://github.com/openshift/machine-config-operator/blob/1e15e3f76c5a050123f38896655c5ca23dafe2ed/pkg/daemon/pivot/utils/run.go#L55  it does return an error if fatal, etc... so we can check for it no?

Reply:
RunExt() calls runExtBackoff() which calls [glog.Fatalf()](https://github.com/openshift/machine-config-operator/blob/master/pkg/daemon/pivot/utils/run.go#L48) in case of any error occurred. [glog.Fatalf()](https://godoc.org/github.com/golang/glog#Fatalf) documentation says that it calls os.Exit() after printing error and stacktrace. So, program will exit at that point.